api-go/internal/graph/graph.go

```diff
@@ -29,7 +27,7 @@ func GenerateAndSaveSVG(ctx context.Context, repo string, merge bool, chartType
 	}
 	defer client.Close()
 
-	graphFunctionUrl := "https://cloudfunction.contributor-graph.com/svg?repo=" + repo
+	graphFunctionUrl := "https://asia-east2-api7-301102.cloudfunctions.net/png" + repo
```

Comment:
Use the original url, which is actually the local balancer URL of the cloud function which adds google cloud armor

Reply:
I don’t know much about Google Cloud, I just want to point it to my newly deployed function.

Reply:
I think you could bind your new function to the load balancer and there are docs talking about it